paddle/framework/framework.proto

```diff
@@ -118,6 +118,38 @@ message LoDTensorArrayDesc {
 
 message ReaderDesc { repeated LoDTensorDesc lod_tensor = 1; }
 
+message MetaDataDesc {
+  enum DataType {
+    BOOL = 1;
+    INT16 = 2;
+    INT32 = 3;
+    INT64 = 4;
+    FP16 = 5;
+    FP32 = 6;
+    FP64 = 7;
+    TENSOR = 8;
+    LOD_TENSOR = 9;
+    SELECTED_ROWS = 10;
+    LOD_TENSOR_ARRAY = 11;
+  }
+  required DataType base_type = 1;
```

Comment:
From what I understand, `base_type` would be the channel type (type of elements in channel) right ?